.github/workflows/release.yml

```diff
@@ -11,6 +11,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          # we need all commit history so we can merge master into develop
+          fetch-depth: 0
```

Comment:
The last steps:

```
git checkout develop
git merge origin/master
git push origin develop
```

need the complete history for `master` and `develop`. `fetch-depth: 0` causes `actions/checkout@v2` to fetch all history.